src/runtime/go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/kata-containers/kata-containers/src/runtime
 
-go 1.21
+// Keep in sync with version in versions.yaml
+go 1.22
```

Comment:
Should we also add a `toolchain` directive to specify its version?

Reply:
Great question - we had quite a debate about this in CAA about whether to just keep the toolchain as a major version e.g. `1.22.0` for us now, or the minor version`1.22.11`, as we didn't want the overhead of bumping the toolchain everything we bumped the go minor version for a security fix, so I avoided that, but it's a point worthy of discussion.

Reply:
I believe the change intended here is just to say "use Go version 1.22" so changing this line to `go 1.22.0` should be sufficient.